docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -339,6 +339,19 @@ installs the collection in the first path defined in :ref:`COLLECTIONS_PATHS`, w
 
 Next, try using the local collection inside a playbook. For examples and more details see :ref:`Using collections <using_collections>`
 
+.. _collections_scm_install:
+
+Installing collections from a git repository
+--------------------------------------------
```

Comment:
You can't install more than one collection from a repo, can you?
```suggestion
Installing a collection from a git repository
---------------------------------------------
```

Reply:
Yes, you can. Either by having multiple directories with galaxy.yml files in the root of the repo or by adding a fragment to the URI.